decidim-core/app/controllers/decidim/devise/omniauth_registrations_controller.rb

```diff
@@ -36,6 +36,8 @@ def create
           on(:error) do |user|
             if user.errors[:email]
               set_flash_message :alert, :failure, kind: @form.provider.capitalize, reason: t("decidim.devise.omniauth_registrations.create.email_already_exists")
+            elsif user.errors[:nickname]
+              set_flash_message :alert, :failure, kind: @form.provider.capitalize, reason: t("decidim.devise.omniauth_registrations.create.nickname_already_exists")
```

Comment:
This error logic is not very good (one single error per form, so two invalid submissions if both fields are incorrect), and I'm not sure it's correct at all. Some tests I made led to `user.errors[:email]` being empty and not `nil` when no errors were found so maybe this error is being displayed everytime if the form has any errors at all?